init/Dockerfile

```diff
@@ -34,7 +34,8 @@ COPY LICENSE /licenses/
 
 # Install dependencies and configmap exporter
 RUN microdnf update -y \
-    && microdnf install wget python3 curl findutils util-linux procps -y \
+    && microdnf install wget python3 python3-pip curl findutils util-linux procps -y \
+    && pip3 install Werkzeug \
```

Comment:
This will result in new init image and for this new image to be used in operator, we need to push this new image to container registry and update its reference in code